#2020/9/23 Patch is upstream, but not officially released. Should be released in libreswan 3.33+
#Description: Force NSS compatibility to allow libreswan to properly autonegotiate IKE encryption
--- a/lib/libswan/ike_alg_encrypt_nss_gcm_ops.c
+++ b/lib/libswan/ike_alg_encrypt_nss_gcm_ops.c
@@ -16,6 +16,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#define NSS_PKCS11_2_0_COMPAT 1
+
 #include "lswlog.h"
 #include "lswnss.h"
 #include "prmem.h"
